src/SwedbankPay.Sdk/Payments/SwishPayments/Payment.cs

```diff
@@ -1,14 +1,16 @@
-﻿using System;
+﻿using SwedbankPay.Sdk.Extensions;
+using System;
+using System.Net.Http;
 using System.Threading.Tasks;
 
-namespace SwedbankPay.Sdk.Payments.Swish
+namespace SwedbankPay.Sdk.Payments.SwishPayments
 {
     public class Payment
```

Comment:
Should this also be renamed to SwishPayment?

Reply:
That it should! Nice catch.